subtests/docker_cli/systemd/docker-build.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=docker-autotest systemd build image subtest. If you see this file on a live system it means something has failed very badly
+
+[Service]
+Type=oneshot
+Environment=NAME=rhel7_p4321
```

Comment:
Since this is going to be altered anyway, how about `NAME=this-will-be-replaced`?